src/components/dashboard/containers/Tables/CampaignsTable.js

```diff
@@ -112,7 +124,7 @@ const getCols = ({
 	},
 	{
 		name: 'fundsDistributedRatio',
-		label: t('PROP_DISTRIBUTED'),
+		label: t('PROP_DISTRIBUTED', { args: ['%'] }),
```

Comment:
If the label has `%` remove it from the `customBodyRender`

Reply:
The translation had a placeholder so this is why I have added it. The translation was displayed as 'Served 0'